src/InvestmentManager.sol

```diff
@@ -358,7 +361,10 @@ contract InvestmentManager is Auth {
             ((maxRedeem(liquidityPool, user)) + trancheTokenPayout).toUint128()
         );
         state.maxWithdraw = state.maxWithdraw + currencyPayout;
-        state.pendingRedeemRequest = remainingRedeemOrder;
+        state.pendingRedeemRequest =
+            state.pendingRedeemRequest > fulfilledRedeemOrder ? state.pendingRedeemRequest - fulfilledRedeemOrder : 0;
```

Comment:
wouldn't you simply use the trancheTokenPayout for that? 
Why do we need the `fulfilledRedeemOrder` variable